mcimx6q-sdb.coffee

```diff
@@ -0,0 +1,34 @@
+deviceTypesCommon = require 'resin-device-types/common'
```

Comment:
Hi @ajlennon , can you follow up with the changes Andrei suggested?

The .coffee name should be <yocto-machine-name>.coffee